ash/src/entry.rs

```diff
@@ -10,7 +10,7 @@ use std::os::raw::c_char;
 use std::os::raw::c_void;
 use std::ptr;
 
-/// Function loader
+/// Hold `static` and `entry` functions
```

Comment:
```suggestion
/// Holds `static` and `entry` functions
```

I don't think "`static` and `entry`" is very helpful to someone who doesn't already understand this type. Maybe a bit more verbosity is justified?

Reply:
Added a bit more verbosity. Library users probably don't - should not - look at the type directly, but it's useful for those implementing their own loader. Couldn't find a proper definition of "entry" functions in Vulkan docs, but the table in `vkGetInstanceProcAddr` should be enough.